central/group/datastore/datastore.go

```diff
@@ -18,9 +18,9 @@ type DataStore interface {
 
 	Add(ctx context.Context, group *storage.Group) error
 	Update(ctx context.Context, group *storage.Group) error
-	Mutate(ctx context.Context, remove, update, add []*storage.Group) error
-	Remove(ctx context.Context, props *storage.GroupProperties) error
-	RemoveAllWithAuthProviderID(ctx context.Context, authProviderID string) error
+	Mutate(ctx context.Context, force bool, remove, update, add []*storage.Group) error
+	Remove(ctx context.Context, force bool, props *storage.GroupProperties) error
+	RemoveAllWithAuthProviderID(ctx context.Context, deleteReq *storage.DeleteByIDWithForce) error
```

Comment:
I'm not sure I like the mixup of the function signatures, where with one we have `force bool, groups...` and a single one using the `storage.DeleteByIDWithForce` as parameter type. 
I'd favor it to be consistent with the signature and change it to 
`RemoveAllWithAuthProviderID(ctx context.Context, force bool, authProviderID string)`.

WDYT?